devices/MAX1704x/MAX1704x.nfproj

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
being very very picky, would be nice as well to have the file renamed with the proper naming convention. In order to do that on Windows: Rename the file with a 2 and the proper case. Commit. Rename by removing the 2, Commit ;-) That's because by default the file system on Windows is case insensitive, so git won't see the change. Once those few files renamed, adjust the sln as well so it won't break on non Windows systems.